flytekit/clis/sdk_in_container/package.py

```diff
@@ -113,6 +136,12 @@ def package(
     object contains the WorkflowTemplate, along with the relevant tasks for that workflow.
     This serialization step will set the name of the tasks to the fully qualified name of the task function.
     """
+    if copy == CopyFileDetection.TEMP_NO_COPY:
```

Comment:
I say we let users specify `--copy none`. It's technically a valid option, but it is not the default.

Reply:
will remove the exception and add a check for conflicts with the fast switch.  But what do you mean by default?  not copying any files is and will continue to be the default for package.